Examples/Modules/qed/schwinger/analysis_schwinger.py

```diff
@@ -99,6 +108,9 @@ def do_analysis(Ex,Ey,Ez,Bx,By,Bz):
         # This first assert only works if a single tile is used in the simulation
         assert(np.array_equal(ele_data,pos_data))
```

Comment:
```suggestion
        print("assert that: np.array_equal(ele_data,pos_data)")
        assert(np.array_equal(ele_data,pos_data))
```
Also, could you explain
```py
# This first assert only works if a single tile is used in the simulation
```
? Would this crash when running parallel?

Reply:
Yes, if there are several boxes or tiles, the electrons and positrons are not in the same position in the data arrays `ele_data` and `pos_data` so the `array_equal` is false (even though there's nothing wrong with the simulation).

With this input script we have an `8*8*8` box and `amr.max_grid_size = amr.blocking_factor = 8` and I think the minimum `tile_size` is 8 by default in amrex so we'll always have 1 box and 1 tile and the test should always work. But the test fails for instance when increasing the box size to 16 in one direction (we have 2 boxes in this case).

Reply:
Otherwise, is it really necessary to print this assert? If it fails, we should have a backtrace of where it failed so it doesn't add any new information, does it?

Reply:
When it fails on TravisCI, you only have access to info you print from your script, so you have to re-run the test locally, which takes some time.

I do not understand why the position of electrons and positrons should be different for a parallel run. Could you please elaborate? How big is the difference? Ideally, we would run the automated test with `mpiexec -np 2`.